backend/src/globals.rs

```diff
@@ -32,20 +33,28 @@ pub static CMD_ARGS: Lazy<Args> = Lazy::new(Args::from_args);
 
 /// The global ValClient for talking to the network.
 pub static CLIENT: Lazy<ValClient> = Lazy::new(|| {
-    let client = ValClient::new(
+    smolscale::block_on(async move {
```

Comment:
Maybe [async_once](https://crates.io/crates/async_once) is more suitable here?

Reply:
I actually don't think so --- I think there are some crates out there that improperly import other languages' intuitions about async and sync being incompatible, "differently-colored" worlds. Rust's async is "polling" rather than "completion" based (basically, async things don't by themselves run off into the background), and interop doesn't come with the same pitfalls as in other langs.

It's completely fine and proper to use things like `block_on` to call async from sync, or `smol::unblock` to call blocking sync from async ^_^

Reply:
Ok awesome, once less dependency to worry about!